packages/backend/src/open_payments/payment_pointer/routes.ts

```diff
@@ -66,10 +68,7 @@ export const listSubresource = async <M extends PaymentPointerSubresource>({
   )
   const result = {
     pagination: pageInfo,
-    result: page.map((item: M) => {
-      item.paymentPointer = ctx.paymentPointer
```

Comment:
We are fetching paymentPointer on the fly (whenever it's missing), this line isn't necessary